packages/t/tmux/package.yml

```diff
@@ -1,8 +1,8 @@
 name       : tmux
-version    : '3.5'
-release    : 23
+version    : 3.5a
```

Comment:
This won't cause another issue like the last version that had a letter in it, will it?

Reply:
Who knows? I'll add quotes.

Reply:
We've had it like that before, a8e2a195d9 but that was before all the tooling changes